### PR TITLE
[KYUUBI #5586][UI] Kyuubi Web UI show current version

### PR DESCRIPTION
**Pull request by zwangsheng**
### _Why are the changes needed?_

As title, let kyuubi web ui header show current version.

### _How was this patch tested?_
- [x] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/master/contributing/code/testing.html#running-tests) locally before make a pull request

npm run dev:
截屏2023-11-01 10 08 56
截屏2023-11-01 10 09 00

### _Was this patch authored or co-authored using generative AI tooling?_

No

### CODE REVIEW

**Comment by zwangsheng** [comment]
As comment in https://github.com/apache/kyuubi/pull/5574#issuecomment-1786433281, PTAL @yaooqinn @pan3793

**Comment by pan3793** [comment]
Does this approach work in release mode?

File: kyuubi-server/web-ui/src/layout/components/aside/index.vue [hunk 1]
```diff
@@ -37,6 +37,11 @@
   const { isCollapse } = storeToRefs(store)
   const router = useRoute()
   const activePath = ref(router.path)
+  /* eslint-disable */
+  // define __APP_VERSION__ in vite.config.ts
+  // @ts-ignore
+  const title = `Apache Kyuubi\n${__APP_VERSION__}`
```
Comment (yaooqinn) [inline kyuubi-server/web-ui/src/layout/components/aside/index.vue:43]: If ``Apache Kyuubi` is necessary here, how about replacing `@/assets/kyuubi.png` with the kyuubi official logo.svg

Comment (pan3793) [reply kyuubi-server/web-ui/src/layout/components/aside/index.vue:43]: I'm afraid the wide logo does not fit the collapsed sidebar

Comment (yaooqinn) [reply kyuubi-server/web-ui/src/layout/components/aside/index.vue]: It seems that SVGs are vector graphics?

**Comment by zwangsheng** [comment]
> Does this approach work in release mode?

Checked in release mode:
1. run `npm run build`
2. cp dist to kyuubi-server resource
3. run kyuubi server

File: kyuubi-server/web-ui/src/layout/components/aside/index.vue (hunk 1 above)
Comment (zwangsheng) [reply kyuubi-server/web-ui/src/layout/components/aside/index.vue]: 截屏2023-11-01 10 09 00
截屏2023-11-01 10 08 56

May this fit expection?

File: kyuubi-server/web-ui/src/layout/components/aside/index.vue [hunk 2]
```diff
@@ -49,19 +55,23 @@
...
-      margin-left: 12px;
-      color: #fff;
-      font-weight: 600;
-      font-size: 14px;
-      font-family: 'Myriad Pro', 'Helvetica Neue', Arial, Helvetica, sans-serif;
+      width: 100px;
+      margin-top: 20px;
+      text-align: center;
+    }
+    .collapse-logo {
```
Comment (yaooqinn) [inline kyuubi-server/web-ui/src/layout/components/aside/index.vue:71]:
```suggestion
    .collapsed-logo {
```

File: kyuubi-server/web-ui/src/layout/components/aside/index.vue [hunk 3]
```diff
@@ -18,8 +18,9 @@
 
 <template>
   <header>
-    <img src="@/assets/kyuubi.png" />
-    <article v-if="!isCollapse">Apache Kyuubi Dashboard</article>
+    <img v-if="!isCollapse" src="@/assets/kyuubi-official-logo.svg" />
+    <img v-else class="collapse-logo" src="@/assets/kyuubi.png" />
```
Comment (yaooqinn) [inline kyuubi-server/web-ui/src/layout/components/aside/index.vue:22]:
```suggestion
    <img v-else class="collapsed-logo" src="@/assets/kyuubi.png" />
```

**Review by yaooqinn — APPROVED** [review]

File: kyuubi-server/web-ui/src/assets/kyuubi-official-logo.svg
Comment (pan3793) [inline kyuubi-server/web-ui/src/assets/kyuubi-official-logo.svg:1]: "official" is redundant

Comment (zwangsheng) [reply kyuubi-server/web-ui/src/assets/kyuubi-official-logo.svg:1]: Done.

**Comment by pan3793** [comment]
Thanks, merged to master/1.8

**Pull request closed without merging**